portal/templates/_partials/header-component/header-component.njk

```diff
@@ -0,0 +1,51 @@
+{% macro render(user) %}
+
```

Comment:
some as above, do we still need this file?

Reply:
i thought I deleted it, will delete